### PR TITLE
Added Footer Dark-05 and Improved Navbar-10

### PR DESCRIPTION
**Pull request by Dibyendu303**
Two issues solved:
1. Added Footer Dark-05. 
Screenshot (1910)

2. #734 Improved Navbar-10
Improve Navbar-10
(My cursor was hovering on "About")

### CODE REVIEW

**Comment by Rajdip019** [comment]
Navbar improvements are not shown in the PR @Dibyendu303. Please check once.

**Comment by Dibyendu303** [comment]
Yeah sorry.. updated now @Rajdip019

File: Footer/src/footer-dark-05.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<div style="margin: 20px;">
```
Comment (Rajdip019) [inline Footer/src/footer-dark-05.html:1]: remove this `style` tag and use tailwind classes instead I guess other than that everything looks fine.

**Comment by Dibyendu303** [comment]
Done @Rajdip019

**Comment by Rajdip019** [comment]
Good job @Dibyendu303

**Pull request merged**